telosnetwork/teloscan
# #530 | Redesign navbar

# Fixes #530 
## Description
This PR overhauls the navbar. it also adds the following changes:
- removes staking altogether
- adds new global variables to `app.scss`: 
    - `--grey-text-color` - muted text color (adapts to light/dark mode)
    - `--border-color` - border for use on tables and buttons etc (adapts to light/dark mode)
    - `--text-color` - default text color for when it needs to be explicitly stated (dark in light mode, white in dark mode)
- adds new high order button component, `OutlineButton`
- adds new file `src/lib/chain-utils.ts` for holding chain info until we switch over to true multi chain support

## Test scenarios
- go to https://deploy-preview-549--teloscan-redesign.netlify.app/
- check out the new header on different screen sizes
    - all elements should be accessible regardless of screen size (e.g. the Network button moves from the top right on desktop to the hamburger menu on mobile)
    - it should look good 😎 
    - bottom part of menu should go away on scroll
    - entire menu should be keyboard accessible
- log in using the connect wallet button
- check out the wallet menu by clicking the wallet button
    - you should be able to view TLOS balance, copy address, view account page, and logout
- change the language
    - the header text should all be localized
- checkout locally and run on **mainnet**
    - you should see TLOS balance fiat value in the account menu after logging in
    - you should see gas and TLOS price in the top left
    - you should not see the testnet indicator on the logo


## Checklist:
